app/src/androidTest/java/com/example/wellfed/RecipeDBTest.java

```diff
@@ -73,6 +76,30 @@ public void testAddRecipe() throws InterruptedException {
             throw new InterruptedException();
         }
 
+        CountDownLatch deleteLatch = new CountDownLatch(1);
+        recipeDB.delRecipe(testRecipe.getId(), (deletedRecipe, success) -> {
+            assertNotNull(deletedRecipe);
+            deleteLatch.countDown();
+        });
+
+        if (!deleteLatch.await(TIMEOUT, SECONDS)) {
+            throw new InterruptedException();
+        }
+
+        CountDownLatch deleteIngredients = new CountDownLatch(2);
+        ingredientDB.deleteIngredient(testIngredient, (deletedIngredient, success) -> {
+            assertNotNull(deletedIngredient);
+            deleteIngredients.countDown();
+        });
+
+        ingredientDB.deleteIngredient(testIngredient2, (deletedIngredient, success) -> {
+            assertNotNull(deletedIngredient);
+            deleteIngredients.countDown();
```

Comment:
We should make an issue to use reference counting similar to what is currently done in StorageIngredient to automatically delete no longer referenced Ingredient objects.

Reply:
An issue has been made for this.